src/Util.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace SilverStripe\Assets;
+
+class Util
+{
+    public static function rewindStream($resource)
```

Comment:
```suggestion
    public static function rewindStream(resource $resource): void
```

Reply:
DONE

Reply:
As above, check use a new `checkIsResource()` method to type check

Reply:
DONE